imports/ui/pages/users/basic_user_information.js

```diff
@@ -119,6 +119,7 @@ Template.basicUserInformation.events({
           icon: "fa-check-circle",
           hideDelay: 6500
         });
+        FlowRouter.go("/welcome");
```

Comment:
After the redirect, the page does not scroll back to top.  jQuery might do the trick.
i.e. `$("html, body").animate({ scrollTop: 0 }, "fast");`